client/src/state/editor/publishBlockMutation.js

```diff
@@ -0,0 +1,44 @@
+import { graphql } from 'react-apollo';
+import gql from 'graphql-tag';
+
+// GraphQL query for saving a specific block
+const mutation = gql`
+mutation PublishBlock($blockId:ID!, $fromStage:VersionedStage!, $toStage:VersionedStage!, $fromVersion:Int!) {
+  copyBlockToStage(Input: {
```

Comment:
@raissanorth @ScopeyNZ @robbieaverill Sorry for tagging you people on this 2 years after the fact, but does anyone recall a reason why we're using `copyBlockToStage` mutation here instead of `publishBlock`?

`copyBlockToStage` doesn't fire the onBeforePublish/onAfterPublish hooks which makes it difficult for developers to extend elements. 

https://github.com/dnadesign/silverstripe-elemental/issues/779

Reply:
Yeah. We had to support a version of SS4 that didn't have that mutation scaffolder I think.

Reply:
That make sense. That's probably out of support by now anyway.  I'll try to dig up when that got added.